metplus/wrappers/compare_gridded_wrapper.py

```diff
@@ -400,14 +400,6 @@ def get_command(self):
         cmd += '-outdir {}'.format(self.outdir)
         return cmd
 
-    def handle_climo_cdf_dict(self):
-        self.add_met_config_dict('climo_cdf', {
-            'cdf_bins': ('float', None, None,
-                         [f'{self.app_name.upper()}_CLIMO_CDF_BINS']),
-            'center_bins': 'bool',
-            'write_bins': 'bool',
```

Comment:
Should we have a line for 'direct_prob' here?

Reply:
The red text shows that these lines were removed in the changes. The new version is found in command_builder.py

Reply:
Ha! Sorry for the confusion!